SPECIFICATION.md

```diff
@@ -113,6 +113,21 @@ LETTER A (U+0041)":
     0.8333,2.5;5.1666,2.5
     0,0;3,9;6,0
 
+In addition to straight lines, polylines can contain circular arc segments.
+This is done by adding a third parameter, called *bulge*, to the coordinate
+pair of the end coordinate of an arc segment. It specifies the included angle of
+the circular arc segment. The bulge is a value from -9 to +9 and represents an
+angle in degrees from -180° to +180°. A positive angle describes a counter
+clockwise arc segment from the start point to the end point, while a negative
```

Comment:
Which point is the start point and the end point? If a coordinate pair is followed by a bulge, is the circle segment from the previous point to the current point, or from the current point to the next point?

Reply:
Should this be clear with this sentence of the explanation?

> This is done by adding a third parameter, called *bulge*, to the coordinate pair of the **end coordinate** of an arc segment.